### PR TITLE
New features added to plot window.

### PR DESCRIPTION
**Pull request by JoeBroesele**
In the context menu of the plot there are new options to select:
* Show/hide the axes from the plot.
* Enable/disable zooming of the time axis.
* Autoscale all value axes (y-axes).

### CODE REVIEW

File: LibreHardwareMonitor/UI/PlotPanel.cs [hunk 1]
```diff
@@ -73,24 +75,50 @@ private ContextMenu CreateMenu()
...
+                new MenuItem("30 min", (s, e) => { TimeAxisZoom(0, 30 * 60); }),
+                new MenuItem("45 min", (s, e) => { TimeAxisZoom(0, 45 * 60); }),
+                new MenuItem("1 h", (s, e) => { TimeAxisZoom(0, 60 * 60); }),
+                new MenuItem("1.5 h", (s, e) => { TimeAxisZoom(0, 1.5 * 60 * 60); }),
+                new MenuItem("2 h", (s, e) => { TimeAxisZoom(0, 2 * 60 * 60); }),
+                new MenuItem("3 h", (s, e) => { TimeAxisZoom(0, 3 * 60 * 60); }),
+                new MenuItem("6 h", (s, e) => { TimeAxisZoom(0, 6 * 60 * 60); }),
+                new MenuItem("12 h", (s, e) => { TimeAxisZoom(0, 12 * 60 * 60); }),
+                new MenuItem("24 h", (s, e) => { TimeAxisZoom(0, 24 * 60 * 60); }) };
+            foreach (MenuItem mi in timeAxisMenuItems)
```
Comment (Phyxion) [inline LibreHardwareMonitor/UI/PlotPanel.cs]: new line before for each for readability, also after the menu.MenuItems.Add below.

File: LibreHardwareMonitor/UI/PlotPanel.cs [hunk 2]
```diff
@@ -73,24 +75,50 @@ private ContextMenu CreateMenu()
...
+            _timeAxesEnableZoom.Changed += (sender, e) =>
+            {
+                _timeAxis.IsZoomEnabled = _timeAxesEnableZoom.Value;
+                InvalidatePlot();
+            };
+
+            MenuItem yAxisMenuItem = new MenuItem("Value Axis");
+            MenuItem[] yAxisMenuItems =
+                { new MenuItem("Autoscale All", (s, e) => { AutoscaleAllYAxes(); }) };
+            foreach (MenuItem mi in yAxisMenuItems)
```
Comment (Phyxion) [inline LibreHardwareMonitor/UI/PlotPanel.cs]: same as previous comment

File: LibreHardwareMonitor/UI/PlotPanel.cs [hunk 3]
```diff
@@ -158,7 +186,7 @@ private PlotModel CreatePlotModel()
             model.Axes.Add(_timeAxis);
             foreach (LinearAxis axis in _axes.Values)
                 model.Axes.Add(axis);
-            model.PlotMargins = new OxyThickness(0);
+            //model.PlotMargins = new OxyThickness(0);
```
Comment (Phyxion) [inline LibreHardwareMonitor/UI/PlotPanel.cs]: Can be removed if unused. Also: what would this default to now if no settings have been set?

File: LibreHardwareMonitorLib/Properties/AssemblyVersion.cs [hunk 4]
```diff
@@ -5,5 +5,5 @@
 
 using System.Reflection;
 
-[assembly: AssemblyVersion("0.8.1.0")]
```
[the remark this answers is not in the document]
Comment (JoeBroesele) [reply LibreHardwareMonitorLib/Properties/AssemblyVersion.cs]: OK. Are You planning to increment the version number at some time?

File: LibreHardwareMonitor/UI/PlotPanel.cs (hunk 1 above)
Comment (JoeBroesele) [reply LibreHardwareMonitor/UI/PlotPanel.cs]: OK. The original code had no new line before the foreach, but I agree it is better with it.

File: LibreHardwareMonitor/UI/PlotPanel.cs (hunk 3 above)
Comment (JoeBroesele) [reply LibreHardwareMonitor/UI/PlotPanel.cs]: OK.
The ScaledPlotModel constructor sets PlotMargins.

**Pull request closed without merging**